OrbitGl/TimeGraph.h

```diff
@@ -102,6 +106,8 @@ class TimeGraph {
 
   void OnLeft();
   void OnRight();
+  void OnShiftLeft();
+  void OnShiftRight();
```

Comment:
I know, you were just following the scheme that was already in place, but maybe it's a good occasion to change these function names to what they are actually doing. So instead of `OnShiftLeft`, maybe something like `SelectPreviousTextBox` or something similar?

Reply:
Or JumpTo...?

Reply:
Done! What do you think?

Reply:
> Done! What do you think?

I don't see a difference? Maybe something got lost?

Reply:
Since I refactor several functions to solve that, I'll make a new PR.

Reply:
:+1: